File: templates/matlab/thermal_system_contribution_bridge.m
function fig = thermal_system_contribution_bridge()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('waterfall', 2508, 'thermal system analysis: contribution waterfall', 'thermal system analysis', 'contribution waterfall');
end

File: templates/python/thermal_system_contribution_bridge.py
"""thermal_system_contribution_bridge: 热学系统贡献瀑布桥（waterfall 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='thermal system analysis: contribution waterfall'):
    return make_template_figure('waterfall', seed=2508, title=title, domain='thermal system analysis', topic='contribution waterfall')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
